docs/source/screening.rst

```diff
@@ -18,35 +18,60 @@ You are asked to make a decision: relevant or irrelevant?
 .. figure:: ../images/project_screening.png
    :alt: ASReview Screening
 
-Click on the decision of choice. A new record is presented to you. While you
-review the records, ASReview LAB continuously improves its understanding of
-your decisions, constantly updating the underlying ordering of the records.
+
+Screening in Oracle mode
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the Oracle mode, unlabeled records are presented to you. Depending on the
+selected strategy it is the most likely relevant record (default
+setting) or based on anothter :ref:project_create:Query Strategy.  
+
+Click on the decision of your choice, and new record is presented to you. While
+you review the next record, a new model is being trained. ASReview LAB
+continuously improves its understanding of your decisions, constantly updating
+the underlying ordering of the records.
+
+Each labeling decision of the user starts the training  of a new model given
+there is no model being trained at that time. When this new model is trained,
+the unseen records' rank order is updated. Training and labeling occur
+asynchronous. With fast models, a new ranking will probably be available
+before the user finished reading the text. With slower models, training
+continues until a new model is trained, and the user can continue screening
+the next record in line (2nd, 3rd, etc.). Therefore, the record shown to the
+user can be the one with the highest relevance score of the second last model
+or the highest-ranked as resulted from the latest model until a new model is
+trained. 
 
 As you keep reviewing documents and providing labels, you will probably see
 fewer relevant records. When to stop screening is left to you. See
 :doc:`progress` for more information on progress monitoring and information on
 when to stop.
 
-.. note::
+.. tip::
 
   If you are in doubt about your decision, take your time as you are the
-  oracle. Based on your input, a new model will be trained in the background.
+  oracle. Based on your input, a new model will be trained, and you do not
+  want to confuse the prediction mode. For the model it may be better to
+  consult others, read the full-text (in case of reviewing abstracts of
+  scientific papers)
 
 Screening in Exploration mode
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-
-In Exploration mode, a blue bar is displayed on top of the record. The blue
-bar indicates whether the record is relevant or irrelevant.
+The Exploration mode is usefull for teaching purposes, because a blue bar is
+displayed on top of the record indicating whether the record has been labeled
+relevant or irrelevant in the dataset. You can make the same labeling
+decission without the need of being the oracle. 
```

Comment:
```suggestion
The Exploration mode is useful for teaching purposes because a blue bar is
displayed on top of the record indicating whether the record has been labeled
relevant or irrelevant in the dataset. You can make the same labeling
decision without the need to be the oracle. 
```